ExtraFiles/common/national_focus/UTA_NF.txt

```diff
@@ -0,0 +1,252 @@
+focus_tree = {
+	id = UTA_focus
+
+	country = {
+		factor = 0
+
+		modifier = {
+			add = 10
+			tag = UTA
+		}
+	}
+	
+	default = no
+	
+	shared_focus = army_effort
+	shared_focus = aviation_effort
+	shared_focus = naval_effort
+	shared_focus = industrial_effort
+
+	focus = {
+		id = UTA_the_yona_doctrine
+		icon = GFX_goal_generic_national_unity
+		x = 21
+		y = 0
+		cost = 10
+		search_filters = {
+			FOCUS_FILTER_POLITICAL
+		}
+		ai_will_do = { factor = 10 }
+		available = {
+			NOT = { has_war = yes }
+			
+		}
+		completion_reward = {
+			create_faction = native_americans
+		}
+	}
+	
+	#Support Mexhican Democracy
+	focus = {
+		id = UTA_support_mexhican_democracy
+		icon = GFX_UTA_support_mexhican_democracy-427
+		x = -3
+		y = 1
+		relative_position_id = UTA_the_yona_doctrine
+		cost = 10
+		search_filters = {
+			FOCUS_FILTER_POLITICAL
+		}
+		prerequisite = { focus = UTA_the_yona_doctrine  }
+		ai_will_do = { factor = 10 }
+		bypass = {
+			MEX = { has_government = democratic }
+			
+		}
+		completion_reward = {
+			MEX = { add_ideas = MEX_strengthen_democracy }
+		}
+	}
+
+	#Call for Elections
+	focus = {
+		id = UTA_call_for_elections
+		icon = GFX_UTA_call_for_elections-427
+		x = 0
+		y = 1
+		relative_position_id = UTA_support_mexhican_democracy
+		cost = 10
+		search_filters = {
+			FOCUS_FILTER_POLITICAL
+		}
+		prerequisite = { focus = UTA_support_mexhican_democracy  }
+		available = { MEX = { democratic > 0.5 } }
+	}
+
+	#Invite Mexhico
+	focus = {
+		id = UTA_invite_mexhico
+		icon = GFX_goal_generic_major_alliance
+		x = 0
+		y = 1
+		relative_position_id = UTA_call_for_elections
+		cost = 10
+		search_filters = {
+			FOCUS_FILTER_POLITICAL
+		}
+		prerequisite = { focus = UTA_call_for_elections  }
+		ai_will_do = { factor = 10 }
+		available = {
+			NOT = { has_war_with = MEX }
+			
+		}
+		bypass = {
+			is_in_faction_with = MEX
+			
+		}
+		completion_reward = {
+			add_to_faction = MEX

```

Comment:
Probably better to use an event, in case MEX is in a faction, a puppet or other cases that auto-adding is undesired.